storage/statedb/database.go

```diff
@@ -310,13 +310,13 @@ func expandNode(hash hashNode, n node) node {
 // NewDatabase creates a new trie database to store ephemeral trie content before
 // its written out to disk or garbage collected.
 func NewDatabase(diskDB database.DBManager) *Database {
```

Comment:
I know it is about this PR, but could you erase this function? There are two identical`NewDatabase()` in database and blockchain package.

Reply:
As we talked, they are different function. We need to refactor the name of functions later.